modin/numpy/array_shaping.py

```diff
@@ -0,0 +1,47 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+"""Module houses array shaping methods for Modin's NumPy API."""
+from modin.error_message import ErrorMessage
+
+
+def ravel(a, order="C"):
+    if order != "C":
+        ErrorMessage.single_warning(
+            "Array order besides 'C' is not currently supported in Modin. Defaulting to 'C' order."
+        )
+    if hasattr(a, "flatten"):
```

Comment:
I believe we discussed using a private variable on modin numpy arrays to check for whether to dispatch to the modin method or default to numpy. Is there a reason we're not taking that approach here, or that we're not defaulting to numpy and instead erroring if there's no `flatten` attribute?

Reply:
Same applies for all other wrapper methods in `array_shaping`, `math`, etc.

Reply:
The check here is to ensure we're operating on a modin.numpy.array. If there's anything that isn't implemented in the modin.numpy.array, it won't be in the modin.numpy namespace - and if someone tries numpy.method(modin.numpy.array) that will either try __array_function__ __array_ufunc__. This check is basically trying to assert that the object (a) is of type modin.numpy.array - I'll make this check more explicit.